### DISCUSSION TITLE
Noise Displayed on the LCD During Startup

### DISCUSSION THREAD

**Discussion by vishnumaiea** [Bodmer/TFT_eSPI#1976] [Q&A - General] [answered]
I tried TFT_eSPI with two different LCDs (**ST7735 128x128** and **ST7789 240x240**). Both of them show **noise/garbage pixels** during the initialization phase. I assume the LCD is showing whatever values in the buffer memory before anything being written onto it.

I tried filling the entire screen with black color just after initialization. But this only helped reducing the duration of the noise on the display. Is there any workaround for this? Is it possible to clear the memory (by writing all black) before the LCD is initialized.

![photo_2022-08-23_09-40-13](https://user-images.githubusercontent.com/10383395/186067814-e49e2a1d-509e-4731-877b-6fd5fe317c8c.jpg)

The exact LCD model shown in the image is **ER-TFT013-1** from BuyDisplay.com.

I am using ESP32 and the default SPI interface with TFT_CS = GPIO15, TFT_DC = GPIO4 and TFT_RST = GPIO16.

**Comment by Bodmer** [comment, marked as answer]
The simplest approach is to keep the backlight LEDs off until the display is initialised.

**Reply by vishnumaiea** [reply]
That seems to be a good trick. Unfortunately I have wired the backlight LEDs directly to the supply on the current PCB. But I will keep this in mind next time. Thank you for the awesome library 🤩
